terminus-ui/cohort-date-range/src/cohort-date-range.component.html

```diff
@@ -1,22 +1,21 @@
-<div>
-  <ts-date-range
-    class="ts-cohort-date-range__date-range"
-    [dateFormGroup]="dateRangeFg"
-    [isDisabled]="!allowCustomDates || isDisabled || disableDateRange"
-    (dateRangeChange)="cohortDateRangeChange($event)"
-  ></ts-date-range>
-  <ts-select
-    label="Select a date range"
-    class="ts-cohort-date-range__select"
-    [isDisabled]="isDisabled"
-    (selectionChange)="selectionChange($event)"
+<ts-date-range
+  class="ts-cohort-date-range__date-range"
+  [dateFormGroup]="dateRangeFormGroup"
+  [isDisabled]="!allowCustomDates || isDisabled || disableDateRange"
+  (dateRangeChange)="cohortDateRangeChange($event)"
+></ts-date-range>
+
+<ts-select
```

Comment:
I guess at some point this would be updated to use `ts-selection-list`?

Reply:
Ahh great callout! I thought these were all already converted. I'll do it in this PR 👍

Reply:
Went in to do this and realized why this wasn't converted.. currently our selection list doesn't support a non-typed version that the standard select does support.